src/teacher.js

```diff
@@ -0,0 +1,27 @@
+
+function submit() {
+    var fName='Student Grade'
+    var in1 = document.getElementById("in1").value;
+    var in2 = document.getElementById("in2").value;
+    var in3 = document.getElementById("in3").value;
+    var in4 = document.getElementById("in4").value;
+    var in5 = document.getElementById("in5").value;
+    var in6 = document.getElementById("in6").value;
+    var in7 = document.getElementById("in7").value;
```

Comment:
For the demo forcing the client to download a file with the information of the student is not necessary. The widgets of the webpage do not have to have proper functionality especially not this kind of functionality

Reply:
I also just noticed that this javascript file shares the same name as a file from the backend. Could you please rename this file to "teacherClientSide.js" to avoid confusion?